### DISCUSSION TITLE
is there a way to full fill the all tab space?

### DISCUSSION THREAD

**Discussion by Sherlock-Holo** [wezterm/wezterm#2953] [Q&A]
in konsole, the tabs will try to full fill the tap space
![image](https://user-images.githubusercontent.com/10096425/212355733-9161ee5d-70dd-4820-82e2-4edca9ea8622.png)

in wezterm doc I don't find the related config, is there a way to full fill the all tab space?

**Comment by wez** [comment]
No, there isn't an option for that

**Reply by Sherlock-Holo** [reply]
will it be supported in the future?

**Reply by wez** [reply]
Maybe? Follow along on:
* https://github.com/wez/wezterm/issues/1914
